pkg/config/remote/service/service.go

```diff
@@ -276,10 +276,33 @@ func (s *Service) ClientGetConfigs(request *pbgo.ClientGetConfigsRequest) (*pbgo
 	if err != nil {
 		return nil, err
 	}
+
+	directorTargets, err := s.uptane.Targets()
+	if err != nil {
+		return nil, err
+	}
+	configPointers, err := executeClientPredicates(request.Client, directorTargets)
+	if err != nil {
+		return nil, err
+	}
+
+	// filter files to only return the configPointers ones
```

Comment:
```suggestion
	// filter files to only return the ones that predicates marked for this client
```

Same nit here, "configPointers" doesn't really tell us pointers to what.